workflow/controller/controller.go

```
@@ -746,12 +748,15 @@ func (wfc *WorkflowController) newPodInformer() cache.SharedIndexInformer {
 			DeleteFunc: func(obj interface{}) {
 				// IndexerInformer uses a delta queue, therefore for deletes we have to use this
 				// key function.
-				key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
+				_, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 				if err != nil {
 					return
 				}
 				// Enqueue the workflow for deleted pod
-				wfc.enqueueWfFromPodLabel(obj, key)
+				err = wfc.enqueueWfFromPodLabel(obj)
```

I think you can totally ignore this error with `_`